framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Templates/CheckRedisPreRequirements.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Linq;
+using Volo.Abp.Cli.ProjectBuilding.Building;
+
+namespace Volo.Abp.Cli.ProjectBuilding.Templates;
+
+public class CheckRedisPreRequirements : ProjectBuildPipelineStep
+{
+    public override void Execute(ProjectBuildContext context)
+    {
+        var modules = context.Files.Where(f => f.Name.EndsWith("Module.cs", StringComparison.OrdinalIgnoreCase));
+        if (modules.Any(module => !module.Content.Contains("Redis:Configuration")))
```

Comment:
There seems to be a problem here. If one of the module files does not have Redis configuration (always), then this *if block* will run, and the `PreRequirements:Redis` extra property will be set. 

Therefore, [it will check that Redis is installed and running](https://github.com/abpframework/abp/blob/6945764746cb8f3b3119a196ad4bb663876c2c5c/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/NewCommand.cs#L138) even it might not need to. 

So, we should either update this line or the line below: https://github.com/abpframework/abp/blob/6945764746cb8f3b3119a196ad4bb663876c2c5c/framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/NewCommand.cs#L138